src/plugins/intel_cpu/tests/unit/shape_inference_test/broadcast_shape_inference.cpp

```diff
@@ -19,15 +19,15 @@ TEST(StaticShapeInferenceTest, BroadcastBidirectionalTest) {
     auto target_shape = std::make_shared<ov::op::v0::Parameter>(element::i32, PartialShape{-1});
     auto broadcast_v3 = std::make_shared<op::v3::Broadcast>(input, target_shape, op::BroadcastType::BIDIRECTIONAL);
 
-    int32_t target_shape_val[] = {1, 16, 50, 50};
+    int32_t target_shape_val[] = {1, 16, 50, 1};
```

Comment:
Is this change because update of update check in broadcast shape inference?

Reply:
It's is not related to any change in the logic, but it makes this test more "bidirectional".